bin/kano-feedback-widget

```diff
@@ -26,8 +26,10 @@ def main():
     # initialise the window
     WidgetWindow()
 
-    # Ignore all SIGTERM signals, app can be closed via a "kill -9".
-    signal.signal(signal.SIGTERM, signal.SIG_IGN)
+    # Protect against all SIGTERM signals
+    if len(sys.argv) > 1 and sys.argv[1] == '--no-kill':
+        # app can be closed via a "kill -9".
```

Comment:
I think this comment is lightly wrong (SIGTERM is not `kill -9`, which it is not possible to ignore).